src/common/relayRequest.types.ts

```diff
@@ -7,7 +7,10 @@ type DeployRequest = EnvelopingTypes.DeployRequestStruct;
 type RelayRequestBody = RelayRequest['request'];
 type DeployRequestBody = DeployRequest['request'];
 
-type EnvelopingRequest = Either<RelayRequest, DeployRequest>;
+type EnvelopingRequest = {
+  request: Either<RelayRequestBody, DeployRequestBody>;
+  relayData: EnvelopingRequestData;
```

Comment:
this type, should not include also EnvelopingDeployData, or something related to the deployment?

Reply:
I'm sorry I don't understand what you mean

Reply:
I'm under the impression that the data is the same for both relay and deploy requests

Reply:
yes you are right, sorry my fault